packages/orchestration/src/examples/kitchen-sink.contract.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * @file Primarily a testing fixture, but also serves as an example of how to
+ *   leverage basic functionality of the Orchestration API with async-flow.
+ */
+import { InvitationShape } from '@agoric/zoe/src/typeGuards.js';
+import { M, mustMatch } from '@endo/patterns';
+import { provideOrchestration } from '../utils/start-helper.js';
+
+/**
+ * @import {Baggage} from '@agoric/vat-data';
+ * @import {Orchestrator} from '@agoric/orchestration';
+ * @import {OrchestrationPowers} from '../utils/start-helper.js';
+ */
+
+/**
+ * Create an account on the local chain and return a continuing offer with
+ * invitations for Delegate, WithdrawRewards, Transfer, etc.
+ *
+ * @param {Orchestrator} orch
+ * @param {undefined} _ctx
+ * @param {ZCFSeat} seat
+ */
+const makeLocalOrchAcctHandler = async (orch, _ctx, seat) => {
+  seat.exit(); // no funds exchanged
+  const agoric = await orch.getChain('agoric');
+  const localAccount = await agoric.makeAccount();
+  // @ts-expect-error asContinuingOffer does not exist on OrchestrationAccountI
+  return localAccount.asContinuingOffer();
+};
+
+/**
+ * Create an account on a Cosmos chain and return a continuing offer with
+ * invitations makers for Delegate, WithdrawRewards, Transfer, etc.
+ *
+ * @param {Orchestrator} orch
+ * @param {undefined} _ctx
+ * @param {ZCFSeat} seat
+ * @param {{ chainName: string }} offerArgs
+ */
+const makeCosmosOrchAcctHandler = async (orch, _ctx, seat, { chainName }) => {
+  seat.exit(); // no funds exchanged
+  mustMatch(chainName, M.string());
+  const remoteChain = await orch.getChain(chainName);
+  const cosmosAccount = await remoteChain.makeAccount();
+  // @ts-expect-error asContinuingOffer does not exist on OrchestrationAccountI
```

Comment:
why not? you did add it to `orchestrationAccountMethods`. Those lists must be the same: https://github.com/Agoric/agoric-sdk/blob/21742bd3d1f6c1ec6693b2378c6eb9efa7ee9999/packages/orchestration/src/utils/orchestrationAccount.js#L11-L22